tests/run-make/broken-pipe-no-ice/rmake.rs

```diff
@@ -5,6 +5,12 @@
 
 //@ ignore-cross-compile (needs to run test binary)
 
+//@ ignore-apple
+// FIXME(jieyouxu): on macOS rustc is still reporting the std broken pipe io error panick but it
```

Comment:
Gentle reminder to create an actual issue for this and add its # to the test so this doesn't get lost in the void.

Reply:
Opened #131436 and referenced that in the test instead.